app/scripts/controllers/modals/editModal.js

```diff
@@ -0,0 +1,123 @@
+'use strict';
+
+angular.module('openshiftConsole')
+  .controller('EditModalController', function($scope, APIService, DataService, QuotaService, LimitRangesService, $uibModalInstance, $filter) {
+
+    var limitRangesVersion = APIService.getPreferredVersion('limitranges');
+    var resourceQuotasVersion = APIService.getPreferredVersion('resourcequotas');
+    var appliedClusterResourceQuotasVersion = APIService.getPreferredVersion('appliedclusterresourcequotas');
+
+    var amountAndUnit = $filter('amountAndUnit');
+    var usageWithUnits = $filter('usageWithUnits');
+    var usageValue = $filter('usageValue');
+
+    var allocatedAmountAndUnit = amountAndUnit($scope.currentCapacity);
+
+    $scope.projectName = JSON.parse($scope.pvc).metadata.namespace;
```

Comment:
Why do we need to call `JSON.parse` here?

Reply:
Ack! Done